tools/deploy_metrics_and_monitoring_resources.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+. $(dirname $0)/../multi-platform-support.sh
+DIR=`pwd`
+k=false
+
+usage() { echo "Usage: $0 [-c <cluster>] -n <namespace>"
+		  echo " When -c is used, Kafka cluster with metrics is deployed."
+		  1>&2; exit 1;}
+
+while getopts "n:c:" o; do
+    case "${o}" in
+        n)  NAMESPACE=${OPTARG}
+        	;;
+        c)  c=true
+            CLUSTER=${OPTARG}
+        	;; 	
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${NAMESPACE}" ] || [[ "$c" == true && -z "${CLUSTER}" ]] ; then
+    usage
+fi
+
+if [ "$c" == true ] ; then
+	# Determine whether CO is deployed in specified NS
+	lines_of_operator=`kubectl get pods -l strimzi.io/kind=cluster-operator -n $NAMESPACE | $WC -l`
+
+	if [ "$lines_of_operator" -lt "2" ]; then
+		echo "Cluster Operator is not deployed!"
+		exit 1
+	fi
+
+	echo "Deploying Kafka "$CLUSTER
+	if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
I do not think you need to do this when you call `sed`. The multiplatform support should set `$SED` to `gsed` which is GNU sed which should be the same as on Linux (unlike the default one which is FreeBSD sed or something like that). So when you use the variables instead of the commends, you should be able to have just a single command in the file for both Linux and MacOS. The downside is that on MAcOS you need to install the GNU versions, but that is in our prerequisites.